packages/Input/src/input.css

```
@@ -0,0 +1,118 @@
+@import "~@igloo-ui/tokens/dist/variables.css";
+@import "~@igloo-ui/tokens/dist/fonts.css";
+
+:root {
+  /* Default */
+  --ids-input-height: var(--space-6);
+  --ids-input-font-family: var(--primary-font-family);
+  --ids-input-font-weight: var(--font-weight-regular);
+  --ids-input-padding: var(--space-3);
+  --ids-input-font-size: var(--font-size-4);
+  --ids-input-line-height: var(--space-4);
+  --ids-input-border-radius: var(--border-radius-sm);
+  --ids-input-border-color: var(--grey-300);
+  --ids-input-background-color: var(--samoyed);
```

Pas besoins d'indiquer `-color` dans le nom de la variable pour les background
```suggestion
  --ids-input-background: var(--samoyed);
```